src/stratum_server.cpp

```diff
@@ -480,7 +481,7 @@ bool StratumServer::on_submit(StratumClient* client, uint32_t id, const char* jo
 		update_auto_diff(client, share.m_timestamp, share.m_hashes);
 
 		// If this share is below sidechain difficulty, process it in this thread because it'll be quick
-		if (!share.m_highEnoughDifficulty) {
+		if (!share.m_highEnoughDifficulty && m_enableFullValidation==false) {
```

Comment:
`&& !m_enableFullValidation`

Reply:
changed to a !m_enableFullValidation instead of compare to false in https://github.com/SChernykh/p2pool/pull/374/commits/164e598677bdf7889328a5c766e81d50911abdee